File: molecule/default/tests/test_default.rb
# frozen_string_literal: true

describe file('/etc/sudoers.d/dmesg') do
  it { should exist }
end

describe file('/root/.bashrc') do
  it { should exist }
end

describe file('/root/.bash_profile') do
  it { should exist }
end

describe file('/etc/mongodb-build-system-id') do
  it { should exist }
end

File: molecule/default/tests/test_default.py
import os

import testinfra.utils.ansible_runner

testinfra_hosts = testinfra.utils.ansible_runner.AnsibleRunner(
    os.environ['MOLECULE_INVENTORY_FILE']
).get_hosts('all')


def test_build_account(host):
    assert host.file("/etc/sudoers.d/dmesg").exists
    assert host.file("/root/.bashrc").exists
    assert host.file("/root/.bash_profile").exists
    assert host.file("/etc/mongodb-build-system-id").exists
